test/04LessonFourTest.js

```diff
@@ -2,40 +2,57 @@
 
 var assert = require('assert');
 
+var arr = ['a','b','c'];
+
 function arrayPop(arr) {
     // should remove the last item in the array, then return the arr
+    arr.pop();
+    return arr;
 
 }
 
 function arrayPush(arr, item1, item2, item3) {
     // should add these items onto the end of the array, then return the array
-
+    arr.push()
+    return arr ;
 }
 
 function arrayShift(arr) {
     // should remove the first item in the array, then return the array
-
+     var shifted = arr.shift
```

Comment:
to remove the first item in an array: arr.shift();
then you want to return the array by: return arr;